model/canvas.cc

```diff
@@ -390,8 +390,13 @@ namespace minsky
       if (i->visible() && lasso.contains(*i))
         selection.wires.push_back(i);
 
-    if (focusFollowsMouse)
+// always copy on lasso selection on X11 systems. for ticket 1189
+#if defined(__linux__)    
```

Comment:
focusFollowsMouse should be enough here. Now I have focusFollowsMouse to be true, and still the selection is not copied by default. It's a weird one.

Reply:
I my case, it worked on and off when focusFollowsMouse was activated. The same thing seems to happen with the wires that are sometimes copied or not.